Makefile

```diff
@@ -26,6 +26,7 @@ install:
 	@echo "Installing cron job..."
 	@install -Dm0644 src/recap.cron $(DESTDIR)$(CRONDIR)/recap
 	@echo "Installing docs..."
+	@install -dm0755 $(DESTDIR)$(DOCDIR)/recap
```

Comment:
This change isn't necessary, because the very next line calls `install` with a `-D` flag, which creates the target directory for you.